features/polyExplorer/src/components/embeddedSankey/embeddedSankey.jsx

```diff
@@ -84,7 +84,7 @@ const EmbeddedSankey = ({ links, groups }) => {
             <ChipGroup
                 chipsContent={sources}
                 defaultActiveChips={
-                    activeSources.length > 1 ? ["all"] : activeSources
+                    activeSources.length > 1 ? ["allChip"] : activeSources
```

Comment:
Have you considered putting this logic directly on the ChipGroup component? In principle, the client shouldn't have to worry about the inclusion or not of the "all" chip, right?

Reply:
But it has to worry about the defaultActiveChips as it needs to set them. Could be that an all chip exists which is not selected on default